man/pam_systemd.xml

```diff
@@ -159,6 +159,20 @@
           </tgroup>
         </table>
 
+        <para>If no session class is specified via either the PAM module option, nor via the
+        <varname>$XDG_SESSION_CLASS</varname> environment variable, the class is automatically chosen, depending on
+        various session parameters, such as the session type (if known), whether the session has a TTY or X11
+        display, and the user disposition. Note that various tools allow setting the session class for newly
+        allocated PAM sessions explicitly and in a fine-grained fashion, by means of the
```

Comment:
Hmm, "in a fine-grained fashion". What's that supposed to mean? If you set the variable, you set the variable.